### PR TITLE
Use https where possible

### PR DESCRIPTION
**Pull request by antleblanc**

### CODE REVIEW

File: google-youtube.html [hunk 1]
```diff
@@ -1,637 +1,637 @@
...
+    <google-youtube
+      videoid="..."
+      height="270px"
+      width="480px"
+      rel="0"
+      start="5"
+      autoplay="1">
+    </google-youtube>
+
+`google-youtube` supports all of the [embedded player parameters](https://developers.google.com/youtube/player_parameters). Each can be set as an attribute on `google-youtube`.
```
Comment (ebidel) [inline google-youtube.html]: why did this entire file change?

Comment (antleblanc) [inline google-youtube.html]: I don't know why this happened to be honest.

I just cloned the repository by using this following command: `git clone`.

Maybe I'm doing something wrong? Any advice?

Thanks.

Comment (ebidel) [inline google-youtube.html]: Github can be fickle with whitespace / line endings. No worries.

**Comment by ebidel** [comment]
LGTM

**Pull request merged**